bin/createTestDB.js

```diff
@@ -30,20 +40,21 @@ const seed = {
     xb: { rescues: 10, rats: 5 },
     ps: { rescues: 10, rats: 5 }
   },
-  timespan: 30 * 86400 * 1000, // 30 days
-  start: Date.UTC(2017,6,1),
+  timespan: DEFAULT_TIMESPAN * SECONDS_IN_DAY * MILLISECONDS,
+  start: Date.UTC(DEFAULT_START_YEAR, DEFAULT_START_MONTH, DEFAULT_START_DAY),
   alpha: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
 }
 
 // predictable simple semi psuedo random generator
 // https://stackoverflow.com/a/19303725/3785845
-function prand (seed = 1, m = 10000) {
+function prand (seed = 1, m = DEFAULT_PRAND_SEED) {
   return function (min, max) {
-    if (max == null) {
+    if (max == null) { // eslint-disable-line
```

Comment:
Type coercion is a big no and if you're going to make an exception I want an explanation of why it's necessary commented in the function.

Reply:
We are checking if the argument max was set.  We cannot check it for 0, because it could be 0 if we are generating -ve numbers.  Rather than check for both undefined and null, I just coerce - this is the one and only time I ever use ==

Reply:
Acceptable, please add a comment clarifying this behaviour.
You could also use ! And eslint won't complain but that's up to you